src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -299,29 +299,29 @@ AttitudeEstimatorQ::Run()
 			}
 		}
 
-		if (_global_pos_sub.updated()) {
-			vehicle_global_position_s gpos;
+		if (_gps_sub.updated()) {
```

Comment:
`gps.eph` is in general not the same as `global_pos.eph`

Reply:
For gating the mag declination update it seems fine (table is extremely coarse). Using the GPS velocity data seems sketchy though. Realistically we're not going to spend much time verifying attq, so I'll move the velocity change to local position.

Reply:
Done.